webrtc-stats.html

```diff
@@ -1749,6 +1750,32 @@
                   mapped to 127.
                 </p>
               </dd>
+              </dd>
+              <dt>
+                <dfn><code>totalAudioEnergy</code></dfn> of type <span class=
+                "idlMemberType"><a>double</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for audio. This value MUST be computed as follows:
+                  for each audio sample sent/received for this object (and
+                  counted by <code><a>totalSamplesSent</a></code> or
+                  <code><a>totalSamplesReceived</a></code>), add the square of
+                  the sample's value divided by the highest-intensity encodable
```

Comment:
Is this too ambiguous? It's meant to mean `square(sample/max)`, not `square(sample)/max`.